Makefile

```diff
@@ -1,30 +1,119 @@
 APP_NAME = event-publisher-proxy
-APP_PATH = components/$(APP_NAME)
-BUILDPACK = eu.gcr.io/kyma-project/test-infra/buildpack-golang:v20220407-4da6c929
-SCRIPTS_DIR = $(realpath $(shell pwd)/../..)/common/makefiles
 
-# fail on lint issues
-override IGNORE_LINTING_ISSUES =
-override ENTRYPOINT = cmd/main.go
+# Image URL to use all building/pushing image targets
+IMG_REGISTRY_PORT ?= 8888
+IMG_REGISTRY ?= op-skr-registry.localhost:$(IMG_REGISTRY_PORT)/unsigned/eventing-images
+IMG ?= $(IMG_REGISTRY)/$(APP_NAME):$(APP_VERSION)
```

Comment:
Is `APP_VERSION` missing or going to be added later?

Reply:
This will come as ENV from build pipeline (i.e. prow), I did not add a default value because we do not have `.version` file in EPP. 

If we had `.version` file we could have done something like [this](https://github.com/kyma-project/eventing-manager/blob/2f2a1b6b12584b159540283c87b46af4b5359ad6/Makefile#L1), but since we are also trying to get rid of `.version` file in other repositories, so I did not add this file.

Reply:
For local dev, we will use the command like `make docker-build IMG=mfaizan21/eventing-publisher-proxy:test1` so we do not need `APP_VERSION` for local development.